lib/data/school_data_helpers.rb

```diff
@@ -0,0 +1,32 @@
+module SchoolDataHelpers
+  def convert_to_point(row)
+    easting  = retrive_easting(row)
+    northing = retrieve_northing(row)
+
+    return nil if easting.blank? || northing.blank?
+
+    coords = Breasal::EastingNorthing.new(
+      easting: easting,
+      northing: northing,
+      type: 'gb'
+    ).to_wgs84
+
+    Bookings::School::GEOFACTORY.point(coords[:longitude], coords[:latitude])
+  end
+
+  def school_types
+    @school_types ||= Bookings::SchoolType.all.index_by(&:edubase_id)
```

Comment:
Wondering if all this should be under `app/models/bookings`, with the shared code in `app/models/bookings/concerns`